Modules/SubModule_Population/xASL_adm_GzipAllFiles.m

```diff
@@ -58,14 +58,14 @@ function xASL_adm_GzipAllFiles(ROOT, bFolder, bUseLinux, pathExternal)
             if ~isempty(PathList)
                 fprintf('\n%s\n',['G-zZzZipping ' num2str(length(PathList)) ' files']);
                 numCores = feature('numcores');
+                glob_pat = join(['"' ROOT '/**/*.nii"']);
                 % Get SuperGzip path
-                PathToSuperGzip = fullfile(pathExternal, 'SuperGZip', 'SuperGZip_Windows.exe');
+                PathToSuperGzip = fullfile(pathExternal, 'SuperGZip', 'super-gunzip.exe');
```

Comment:
Can you provide a solution for Linux, macOS, and Windows? Only Windows is also fine, but that would only serve part of the users.

I would still not call it `super-gunzip.exe` if we also (or even more frequently) use it for zipping; fine to rename to `super-zip.exe`?

Reply:
Sounds good to me. I'll keep calling it super-gunzip in the original repo & its release page. But when copying over to ExploreASL, will change name to just super-zip.

Reply:
How does it perform compared to the default Linux parallel zipping algorithm, called `p7zip`?

Reply:
I would still say that that is weird. No zip package calls itself unzip as primary name right? 7zip not 7unzip. Winrar not WinUnrar. Etc.